tests/dummy-wallet-app/src/index.tsx

```diff
@@ -28,10 +39,22 @@ function TransportProvider({ children }: PropsWithChildren<object>) {
     return transport;
   }
 
+  const eventHandlers = useMemo<EventHandlers>(() => {
+    return {
+      "event.custom.test": param => {
+        console.log("event handler param:", param);
+      },
+    };
+  }, []);
```

Comment:
Not sure if we should keep this here or create an example like the custom logger in the wallet-api folder of common